### ISSUE TITLE
Fish_eye Camera Issue

### ISSUE THREAD

**Issue by asifsid-32** [lgsvl/simulator#1039]
Hii LGSVL Team,

I am still facing the issue with the brightness of fish_eye camera on my ego vehicle. What can we do to improve this ?
![issue_new](https://user-images.githubusercontent.com/69885768/94804252-24e26b80-0408-11eb-8196-c00c90f73838.png)

**Comment by EricBoiseLGSVL** [comment]
@asifsid-32 You will need to set a fixed exposure in HDRP settings until we can work on this issue.  Since exposure is calculated async, it varies for camera rotation and gives different values.  We plan on a fix for this soon but editing HDRP will fix this now.  It will be a little dark at night.

**Comment by asifsid-32** [comment]
> @asifsid-32 You will need to set a fixed exposure in HDRP settings until we can work on this issue.  Since exposure is calculated async, it varies for camera rotation and gives different values.  We plan on a fix for this soon but editing HDRP will fix this now.  It will be a little dark at night.  See #913 

Hii @EricBoiseLGSVL is there anyway I can get the image from my fisheye camera sensor directly from the unity editor as I get from save_camera_image.py ??

**Comment by EricBoiseLGSVL** [comment]
@asifsid-32 Yes, we are posting the camera image to the visualizer, please look at this code in the UIManager and sensors to see how to access the Unity RenderTexture.

**Comment by asifsid-32** [comment]
> @asifsid-32 You will need to set a fixed exposure in HDRP settings until we can work on this issue. Since exposure is calculated async, it varies for camera rotation and gives different values. We plan on a fix for this soon but editing HDRP will fix this now. It will be a little dark at night. See #913

Hii @EricBoiseLGSVL as you said in https://github.com/lgsvl/simulator/issues/945 
"In the meantime you can set Exposure to fixed and 0 in PostProcessingVolume.prefab to eliminate the issue. It's in Assets/Prefabs/"

I am not able to find Exposure in PostProcessingVolume.prefab. Is it somewhere else that i have to look? Can you help with this ?

**Comment by EricBoiseLGSVL** [comment]
Type in PostProcessingVolume in the Unity editor project panel.  It is in Assets/Prefabs.  The volume is Exposure.  It needs set to Fixed and 0.

**Comment by asifsid-32** [comment]
> Type in PostProcessingVolume in the Unity editor project panel.  It is in Assets/Prefabs.  The volume is Exposure.  It needs set to Fixed and 0.

Hii @EricBoiseLGSVL so if i change this in unity will the changes get reflected for the cameras which are there in json script ?

**Comment by EricBoiseLGSVL** [comment]
Yes, this is a global post process volume

**Comment by asifsid-32** [comment]
> Yes, this is a global post process volume

Thanks @EricBoiseLGSVL it helped :)
